main.go

```diff
@@ -267,120 +253,15 @@ func main() {
-			Mutating:    false,
-		},
-		{
-			Registrator: (&policiesv1.ClusterAdmissionPolicy{}).SetupWebhookWithManager,
-			Name:        "mutate-clusteradmissionpolicies.kubewarden.dev",
-			RulesWithOperations: []admissionregistrationv1.RuleWithOperations{
-				{
-					Operations: []admissionregistrationv1.OperationType{
-						admissionregistrationv1.Create,
-						admissionregistrationv1.Update,
-					},
-					Rule: admissionregistrationv1.Rule{
-						APIGroups:   []string{policiesv1.GroupVersion.Group},
-						APIVersions: []string{policiesv1.GroupVersion.Version},
-						Resources:   []string{"clusteradmissionpolicies"},
-					},
-				},
-			},
-			WebhookPath: "/mutate-policies-kubewarden-io-v1-clusteradmissionpolicy",
-			Mutating:    true,
-		},
-		{
-			Registrator: (&policiesv1.ClusterAdmissionPolicy{}).SetupWebhookWithManager,
-			Name:        "validate-clusteradmissionpolicies.kubewarden.dev",
-			RulesWithOperations: []admissionregistrationv1.RuleWithOperations{
-				{
-					Operations: []admissionregistrationv1.OperationType{
-						admissionregistrationv1.Create,
-						admissionregistrationv1.Update,
-					},
-					Rule: admissionregistrationv1.Rule{
-						APIGroups:   []string{policiesv1.GroupVersion.Group},
-						APIVersions: []string{policiesv1.GroupVersion.Version},
-						Resources:   []string{"clusteradmissionpolicies"},
-					},
-				},
-			},
-			WebhookPath: "/validate-policies-kubewarden-io-v1-clusteradmissionpolicy",
-			Mutating:    false,
-		},
-		{
-			Registrator: (&policiesv1.AdmissionPolicy{}).SetupWebhookWithManager,
-			Name:        "mutate-admissionpolicies.kubewarden.dev",
-			RulesWithOperations: []admissionregistrationv1.RuleWithOperations{
-				{
-					Operations: []admissionregistrationv1.OperationType{
-						admissionregistrationv1.Create,
-						admissionregistrationv1.Update,
-					},
-					Rule: admissionregistrationv1.Rule{
-						APIGroups:   []string{policiesv1.GroupVersion.Group},
-						APIVersions: []string{policiesv1.GroupVersion.Version},
-						Resources:   []string{"admissionpolicies"},
-					},
-				},
-			},
-			WebhookPath: "/mutate-policies-kubewarden-io-v1-admissionpolicy",
-			Mutating:    true,
-		},
-		{
-			Registrator: (&policiesv1.AdmissionPolicy{}).SetupWebhookWithManager,
-			Name:        "validate-admissionpolicies.kubewarden.dev",
-			RulesWithOperations: []admissionregistrationv1.RuleWithOperations{
-				{
-					Operations: []admissionregistrationv1.OperationType{
-						admissionregistrationv1.Update,
-					},
-					Rule: admissionregistrationv1.Rule{
-						APIGroups:   []string{policiesv1.GroupVersion.Group},
-						APIVersions: []string{policiesv1.GroupVersion.Version},
-						Resources:   []string{"admissionpolicies"},
-					},
-				},
-			},
-			WebhookPath: "/validate-policies-kubewarden-io-v1-admissionpolicy",
-			Mutating:    false,
-		},
+func setupWebhooks(mgr ctrl.Manager, deploymentsNamespace string) error {
```

Comment:
Do we still need this function?

Looking into the [`policiesv1.SetupPolicyServerWebhookWithManager`](https://github.com/kubewarden/kubewarden-controller/blob/6361dc9a26885d4c14923665b57eba0b3a444121/pkg/apis/policies/v1/policyserver_webhook.go#L38-L54) I see there are comments about that being a function required by rafa's library

Reply:
I'll check the comment in that function. However, we need this function to make linter happy. It was complaining about the complexity of the man function when I added the calls to setup the webhook in the manager.

Reply:
@flavio the comment was referring to the removal of the workaround to return a function, since the former library was taking a setup function as first class citizen parameter when setting up the webhook. The Setup function as refactored in this PR.

Reply:
> @flavio the comment was referring to the removal of the workaround of returning a function, since the former library was taking a setup function as first class citizen parameter when setting up the webhook. The PolicyServer webhook setup function was refactored in this PR.

Yes, I've noticed that when I've read the code and already fix it.